platform/android/java/nativeSrcsConfigs/CMakeLists.txt

```diff
@@ -17,3 +17,5 @@ target_include_directories(${PROJECT_NAME}
         SYSTEM PUBLIC
         ${GODOT_ROOT_DIR}
         ${GODOT_ROOT_DIR}/modules/gdnative/include)
+
+add_definitions(-DUNIX_ENABLED -DTOOLS_ENABLED)
```

Comment:
That looks weird, but given the comment at the top I guess this is just added to allow Android Studio to consider code in those `#ifdef`s as part of the build config?

Reply:
Yep!